### PR TITLE
Feature: Implemented Orders Table for PayPal Handler

### LINKED ISSUE #7674: [PayPal] Implement the Orders table
### Short description of current behavior

MindsDB integrates with PayPal, so you can access PayPal data and make relevant predictions with MindsDB.

## Resources

Familiarize yourself with the existing PayPal integration: https://github.com/mindsdb/mindsdb/tree/staging/mindsdb/integrations/handlers/paypal_handler

## Steps to Follow

Currently, the PayPal integration with MindsDB accesses the Payments table.

The task is to extend [this implementation](https://github.com/mindsdb/mindsdb/tree/staging/mindsdb/integrations/handlers/paypal_handler) to include the Orders table.

Please provide screenshots or a short video in the PR to show successful test results.

## :loudspeaker: Additional rewards :trophy:

Each integration extension issue brings 15+ points for SWAG and entry into the draw for a :point_down: :
-  :computer:  [Razer Blade 16 Laptop](https://www.razer.com/gaming-laptops/Razer-Blade-16/RZ09-0483SEJ3-R3U1)
- :shirt: :bear:  [MindsDB Swag](https://store.covver.io/mindsdb/collections/swag-collection)

:information_source: For more info check out https://mindsdb.com/hacktoberfest/ :point_left: 

### Video or screenshots

_No response_

### Expected behavior

_No response_

### How to reproduce the error

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by shaharsh624**
## Description

Added orders table implementation for PayPal handler.

Fixes #7674 

## Type of change

- [x] ⚡ New feature (non-breaking change which adds functionality)

## Verification Process

To ensure the changes are working as expected:

 - [x]   Test Location: https://github.com/shaharsh624/mindsdb/blob/staging/mindsdb/integrations/handlers/paypal_handler/paypal_tables.py

## Checklist:

- [x] My code follows the style guidelines(PEP 8) of MindsDB.
- [x] I have appropriately commented on my code, especially in complex areas.
- [x] Necessary documentation updates are either made or tracked in issues.
- [x] Relevant unit and integration tests are updated or added.

### CODE REVIEW

**Comment by ZoranPandovski** [comment]
recheckcla

**Comment by mindsdbadmin** [comment]
**CLA Assistant Lite bot:** <br/>Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that you sign our [Contributor License Agreement](https://github.com/mindsdb/mindsdb_native/blob/stable/assets/contributions-agreement/individual-contributor.md) before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.<br/>
   - - -
   I have read the CLA Document and I hereby sign the CLA
   - - -
   <sub>You can retrigger this bot by commenting **recheck** in this Pull Request</sub>

File: mindsdb/integrations/handlers/paypal_handler/paypal_tables.py [hunk 1]
```diff
@@ -52,3 +52,59 @@ def get_payments(self, **kwargs) -> List[Dict]:
         connection = self.handler.connect()
         payments = paypalrestsdk.Payment.all(kwargs, api=connection)
         return [payment.to_dict() for payment in payments['payments']]
+
+
+import paypalrestsdk
```
Comment (ZoranPandovski) [inline mindsdb/integrations/handlers/paypal_handler/paypal_tables.py:57]: Let's add the imports on top

File: mindsdb/integrations/handlers/paypal_handler/paypal_tables.py [hunk 2]
```diff
@@ -52,3 +52,59 @@ def get_payments(self, **kwargs) -> List[Dict]:
...
+
+from mindsdb_sql.parser import ast
+from mindsdb.integrations.libs.api_handler import APITable
+
+from mindsdb.integrations.handlers.utilities.query_utilities import SELECTQueryParser, SELECTQueryExecutor
+
+
+class OrdersTable(APITable):
+
+  def select(self, query: ast.Select) -> pd.DataFrame:
```
Comment (ZoranPandovski) [inline mindsdb/integrations/handlers/paypal_handler/paypal_tables.py:69]: Please check if this is valid?

**Review by ZoranPandovski — CHANGES_REQUESTED** [review]

**Pull request closed without merging**